node/src/BaseClient.ts

```diff
@@ -2338,9 +2338,9 @@ export class BaseClient {
      * ```
      */
     public async lset(
-        key: string,
+        key: GlideString,
         index: number,
-        element: string,
+        element: GlideString,
     ): Promise<"OK"> {
         return this.createWritePromise(createLSet(key, index, element));
```

Comment:
Use string decoder hardcoded to ensure that OK isn't encoded

Reply:
Updated.